lib/index.js

```diff
@@ -76,11 +76,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         }
 
         const {actor} = req.user;
-        await brAuthnToken.set({
+        const result = await brAuthnToken.set({
           actor, account, email, type, hash, clientId,
           authenticationMethod, requiredAuthenticationMethods
         });
-        res.status(204).end();
+        res.json(result);
```

Comment:
We can't do this for `nonce`, it returns the plain text challenge -- which would enable anyone to authenticate using a nonce token without having to access email/whatever notification service it should depend on.

Reply:
Oh, if github gives me more context here, I see we're not doing it for nonce. It is a bit weird to return 204 in one case and 200 in another. @davidlehn can we make this more clean somehow?